google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -1435,16 +1453,52 @@ def routines token: nil, max: nil, filter: nil
         #
         # @!group Data
         #
-        def query_job query, params: nil, types: nil, external: nil, priority: "INTERACTIVE", cache: true, table: nil,
-                      create: nil, write: nil, dryrun: nil, standard_sql: nil, legacy_sql: nil, large_results: nil,
-                      flatten: nil, maximum_billing_tier: nil, maximum_bytes_billed: nil, job_id: nil, prefix: nil,
-                      labels: nil, udfs: nil
+        def query_job query,
+                      params: nil,
+                      types: nil,
+                      external: nil,
+                      priority: "INTERACTIVE",
+                      cache: true,
+                      table: nil,
+                      create: nil,
+                      write: nil,
+                      dryrun: nil,
+                      standard_sql: nil,
+                      legacy_sql: nil,
+                      large_results: nil,
+                      flatten: nil,
+                      maximum_billing_tier: nil,
+                      maximum_bytes_billed: nil,
+                      job_id: nil,
+                      prefix: nil,
+                      labels: nil,
+                      udfs: nil,
+                      create_session: nil,
+                      session_id: nil
```

Comment:
Dang, that really makes it clear how many arguments this method takes... 😳
I wonder if we should sort arguments by name in long lists like this.

Reply:
I know... I hated to add to the list, but I feel these session args really deserve it. See the example for usage.

I'm totally open to sorting. Personally I really like alphabetical order. The established pattern in these libraries has been order of importance initially, then order in which each element was added, since the beginning.

Reply:
Resolving for now, as we've opened https://github.com/googleapis/google-cloud-ruby/issues/16026 to further discussion.